examples/utils/example-utils/src/modelLoader/modelContainerRuntimeFactoryWithAttribution.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IContainer,
+	IContainerContext,
+	IRuntime,
+	IRuntimeFactory,
+} from "@fluidframework/container-definitions";
+import { IContainerRuntimeOptions, ContainerRuntime } from "@fluidframework/container-runtime";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { NamedFluidDataStoreRegistryEntries } from "@fluidframework/runtime-definitions";
+import {
+	mixinAttributor,
+	createRuntimeAttributor,
+	IProvideRuntimeAttributor,
+} from "@fluid-experimental/attributor";
+import { FluidObject } from "@fluidframework/core-interfaces";
+import { makeModelRequestHandler } from "./modelLoader";
+
+const containerRuntimeWithAttribution = mixinAttributor(ContainerRuntime);
```

Comment:
I think for now it would be better to not lift this to the higher package--we only have one example that uses a container runtime with mixed-in attribution. This is also code that we want consumers to look at in order for them to understand how to make attribution work (they need to write this!), which they probably won't do if it's in a utils package.